install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2562,8 +2562,13 @@ clustermesh:
       digest: ${CLUSTERMESH_APISERVER_DIGEST}
       useDigest: ${USE_DIGESTS}
       pullPolicy: "${PULL_POLICY}"
+
     # -- TCP port for the clustermesh-apiserver health API.
     healthPort: 9880
+    # readinessProbe:
```

Comment:
Nit. I wonder if it could make sense to uncomment `readinessProbe:` (adding `{}` at the end), so that it is possible to add a comment, and it appears as part of the readme. Same for kvstoremesh.

Reply:
updated :+1: